numpy/linalg/lapack_lite/f2c.h

```diff
@@ -352,7 +354,7 @@ extern void s_copy(char *, char *, ftnlen, ftnlen);
 extern int s_paus(char *, ftnlen);
 extern integer s_rdfe(cilist *);
 extern integer s_rdue(cilist *);
-extern integer s_rnge(char *, integer, char *, integer);
+extern int s_rnge(char *, int, char *, int);
```

Comment:
Any idea what defines this function? Perhaps the definition should change. (`f2c.c`?)

Reply:
The definition is like this in `f2c.c`, and the function is unused.